File: src/main/java/by/forward/forward_system/core/dto/messenger/v3/DistributionLogDto.java
package by.forward.forward_system.core.dto.messenger.v3;

import by.forward.forward_system.core.dto.rest.users.UserDto;
import by.forward.forward_system.core.enums.DistributionStatusType;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class DistributionLogDto {
    private Long id;
    private Long orderId;
    private List<DistributionItemLogDto> items;
    private DistributionStatusType statusType;
    private String statusTypeRus;
    private UserDto createdBy;
    private String createdAt;
}

File: forward-system-frontend/src/core/dto/DistributionLogDto.ts
import type {DistributionItemLogDto} from "@/core/dto/DistributionItemLogDto.ts";
import type {UserDto} from "@/core/dto/UserDto.ts";

export interface DistributionLogDto {
    id: number,
    orderId: number
    items: DistributionItemLogDto[],
    statusType: string,
    statusTypeRus: string,
    createdBy: UserDto,
    createdAt: string,
    startTimeAt: string,
}